deft.js

```diff
@@ -115,29 +115,17 @@ deft.normalizeFilePaths = function(files) {
  * @return {string}
  *
  * @examples
- * deft.getUrl('lodash.js', ['lodash/lodash', 'lodash.js']);
- * // => 'https://raw.githubusercontent.com/lodash/lodash/master/lodash.js'
+ * deft.getInformationUrl('lodash.js', ['lodash/lodash', 'lodash.js']);
+ * // => 'https://api.github.com/repos/lodash/lodash/contents/lodash.js?ref=master'
  *
- * deft.getUrl('underscore.js', ['jashkenas/underscore', '1.6.0', 'underscore.js']);
```

Comment:
Well my deft.json test file contains a download for http://code.angularjs.org/1.2.5/angular-1.2.5.zip and it still works, this function wasn't used for anything else than gh downloads.